### PR TITLE
fix(DataTable): update gap space between content and indicator

### PR DESCRIPTION
**Pull request by laraharrow**
Thank so much for contributing to @looker/components!

The CI & [README](https://github.com/looker-open-source/components/blob/main/README.md) should help to make most of our code standards clear

Before requesting review from another developer please delete the lines above and summarize your change.

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [ ] a11y impact (FUTURE: Make aXe pass req'd for CI ✅)

#### Image snapshots (choose one)

  - [x] yes
  - [ ] not applicable

#### Documentation updated (choose one)

  - [ ] yes
  - [x] not applicable

### CODE REVIEW

File: packages/components/src/DataTable/stories/useIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    </DataTableCell>
+  </DataTableItem>
+))
+
+const Template: Story<DataTableProps> = ({ ...args }) => {
+  return <DataTable {...args} />
+}
+
+export const UseIndicator = Template.bind({})
+UseIndicator.args = {
```
Comment (dbchristopher) [inline packages/components/src/DataTable/stories/useIndicator.tsx:78]: Just for the sake of preventing future confusion, I'd probably just rename the story and filename to simply `Indicator`. The convention for `use...` usually indicates a custom hook. But I think in this case it's simply indicating that the indicator prop is being used?

**Review by dbchristopher — CHANGES_REQUESTED** [review]
The code looks good! I just left a comment about naming convention that I think would be good to update so as not to cause any confusion in the future

File: packages/components/src/DataTable/stories/useIndicator.tsx (hunk 1 above)
Comment (laraharrow) [reply packages/components/src/DataTable/stories/useIndicator.tsx:78]: oh good point!

**Review by dbchristopher — DISMISSED** [review]
Looks good! 👍

**Review by dbchristopher — APPROVED** [review]

**Pull request merged**